dlk/python/dlk/templates/src/func/impl/generic/quantized_conv2d_kn2row.cpp

```diff
@@ -26,20 +26,24 @@ limitations under the License.
 
 namespace {
 
+using kn2row_kernel_t = TensorView<QUANTIZED_PACKED_KERNEL, MemoryLayout::HWNC>;
+
 // kernel format converter
 // ohwi : oc kh kw ic, hwoi: kh kw oc ic
-void quantized_ohwi_to_hwoi(const QUANTIZED_PACKED_KERNEL ohwi[], QUANTIZED_PACKED_KERNEL hwoi[], const struct binary_convolution_parameters& p) {
+void quantized_ohwi_to_hwoi(const kernel_t& ohwi, const kn2row_kernel_t& hwoi, const struct binary_convolution_parameters& p) {
```

Comment:
Is it possible to move this function to `tensor_convert.h`?

Reply:
I mean, I'd like to hear your idea. If you think it's better to keep it here, it's OK.

Reply:
Implemented new convert_tensor overload.

Reply:
thanks!